api/mqs/ironmq.go

```diff
@@ -68,34 +69,38 @@ func NewIronMQ(url *url.URL) *IronMQ {
 	}
 
 	var queueName string
-	if url.Path != "" {
-		queueName = url.Path
+	if len(url.Path) > 1 {
+		// Drop leading '/'.
+		queueName = url.Path[1:]
```

Comment:
Hopefully we can get this fix in soon. Waiting on #599 at the moment, then the rest of this PR will follow in small portions.